command/s3_put.go

```diff
@@ -162,11 +163,14 @@ func (s3pc *s3put) expandParams(conf *model.TaskConfig) error {
 		return errors.WithStack(err)
 	}
 
-	s3pc.skipMissing, err = s3pc.Optional.Bool()
-	if err != nil {
-		return errors.WithStack(err)
+	if s3pc.Optional != "" {
+		s3pc.skipMissing, err = strconv.ParseBool(s3pc.Optional)
+		if err != nil {
+			return errors.WithStack(err)
+		}
+	} else {
+		s3pc.skipMissing = false
```

Comment:
I'm aware this seems redundant, but this value can get set elsewhere (only in tests?).